src/components/Greeting.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchGreeting } from '../feature/greetingSlice.jsx';
+
+const Greeting = () => {
+  const { greeting } = useSelector((store) => store.greetings);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+
+  const handler = () => {
+    dispatch(fetchGreeting());
+  };
+
+  console.log(greeting);
```

Comment:
- [ Optional ] I think it would be better if you remove this `console.log` statement from your `Greeting` component, the `console.log` statements if left in your code slow your code and at times can log out very sensitive information from your application e.g `passwords`, `API` keys. Kindly remove this line of code from your code base.